### DISCUSSION TITLE
Replacing certain characters in the upstream version with others

### DISCUSSION THREAD

**Discussion by Preisschild** [renovatebot/renovate#19167] [Request Help]
Hello

I currently have a use case where I need to replace the semver seperator dots with another character, as the config file in which the version is specified does not allow dots.

I thought this would be easily possible using`extractVersionTemplate` and the `replace` handlebars helper like this:
```
"extractVersionTemplate": "{{{ replace '.' '_' version }}}" 
```

Unfortunately, renovate seems to ignore this and tries to replace the `_` in the currentVersion with dots.

To further illustrate what I'm trying to do, I've created a demo repo [here](https://github.com/youniqx/renovate-replaceversioncharacters-demo).

The `test.yaml` file contains the version. Renovate created [this PR](https://github.com/youniqx/renovate-replaceversioncharacters-demo/pull/1).

Is this currently possible, or would this be a new feature?

**Comment by rarkins** [comment]
I don't think this is currently possible
